integrations/elevenlabs.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: integration
+name: Elevenlabs
+description: ElevenLabs Text-to-Speech components for Haystack.
+authors:
+    - name: Andy
+      socials:
+        github: andychert
+        twitter: andychert
+pypi: https://pypi.org/project/elevenlabs-haystack/
+repo: https://github.com/andychert/elevenlabs-haystack
+type: TTS
+report_issue: https://github.com/andychert/elevenlabs-haystack/issues
+logo: https://private-user-images.githubusercontent.com/12028621/243114574-21267d89-5e82-4e7e-9c81-caf30b237683.png
+version: Haystack 2.0
+toc: true
+---
+
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+This package provides Haystack components for integrating ElevenLabs Text-to-Speech services into your Haystack pipelines.
+
+## Installation
+
+```bash
+pip install elevenlabs-haystack
+```
+
+## Usage
+
+```python
+from haystack import component, Pipeline
+from haystack.utils import Secret
+
+from elevenlabs_haystack import ElevenLabsTextToSpeech
+
+@component
+class WelcomeTextGenerator:
+  """
+  A component generating personal welcome message and making it upper case
+  """
+  @component.output_types(welcome_text=str, note=str)
+  def run(self, name:str):
+    return {"welcome_text": ('Hello {name}, welcome to Haystack!'.format(name=name)).upper(), "note": "welcome message is ready"}
+
+text_pipeline = Pipeline()
+text_pipeline.add_component(name="welcome_text_generator", instance= WelcomeTextGenerator())
+text_pipeline.add_component(
+    instance=ElevenLabsTextToSpeech(
+        elevenlabs_api_key=Secret.from_token("elevenlabs_api_key"),
+        output_folder="audio_files",
+        voice_id="K8lgMMdmFr7QoEooafEf",
+        aws_s3_bucket_name="mybucket",
+        aws_access_key_id=Secret.from_token("aws_access_key_id"),
+        aws_secret_access_key=Secret.from_token("aws_secret_access_key"),
+        aws_region_name="us-east-1",
```

Comment:
Can you provide more information/related links about how to get the access keys and why we need these fields?